miio/integrations/vacuum/mijia/g1vacuum.py

```diff
@@ -373,3 +373,18 @@ def consumable_reset(self, consumable: G1Consumable):
     def set_fan_speed(self, fan_speed: G1FanSpeed):
         """Set fan speed."""
         return self.set_property("fan_speed", fan_speed.value)
+
+    @command()
+    def fan_speed_presets(self) -> FanspeedPresets:
+        """Retrieve predefined fan speeds, see VacuumInterface for details."""
```

Comment:
```suggestion
        """Return available fan speed presets."""
```
We want to make a difference between fan speed and fan speed preset, the former is something that not all vacuums support.